src/patternfly/components/Card/card.scss

```diff
@@ -146,6 +161,19 @@
 
     padding: 0;
   }
+
+  &.pf-m-toggle-right {
+    --pf-c-card__header-toggle--MarginRight: var(--pf-c-card__header--m-toggle-right--toggle--MarginRight);
+    --pf-c-card__header-toggle--MarginLeft: var(--pf-c-card__header--m-toggle-right--toggle--MarginLeft);
```

Comment:
The right toggle overlaps with the actions element.

<img width="1021" alt="Screen Shot 2021-03-10 at 6 39 05 PM" src="https://user-images.githubusercontent.com/35148959/110718637-00699700-81d1-11eb-9914-1ce9e54388c6.png">

There is a negative margin on actions that I don't think needs to be there. I dug into it and I think I accidentally added that margin here - https://github.com/patternfly/patternfly/pull/3586/files#diff-f77b58a7cf3d9657d084dec7fcc8ac27020c93e9dedea7d718cc122ccf48cd83R135

I created an issue to sort that out - https://github.com/patternfly/patternfly/issues/3924